### ISSUE TITLE
Add issues on logging SQL errors or SQLi payloads

### ISSUE THREAD

**Issue by jmanico** [OWASP/CheatSheetSeries#1127]
Suggest a new logging vocab entry to detect SQL errors (potential SQLi)

**Comment by mackowski** [comment]
FYI @thatsjet

**Comment by thatsjet** [comment]
ack

**Comment by jmanico** [comment]
Hey @thatsjet are you still into this?

**Comment by thatsjet** [comment]
Yes! I have a bunch of updates ready to submit for PR soon. Have also
written some libraries for common languages that implement it. We're using
this heavily at current company. I'll put cycles into this this week.
Thanks for the nudge!
